roles/nf-core/defaults/main.yml

```diff
@@ -34,7 +34,7 @@ pipelines:
       - CanFam3.1
       - WBcel235
   - name: methylseq
-    release: 2.3.0
+    release: 3.0.0
```

Comment:
Do we want to make our own repo for this?

Reply:
Maybe it's better to wait until we need to make our modifications to this pipeline? Otherwise it just increases the maintenance burden without giving us much in return.